core/src/test/java/com/airbnb/aerosolve/core/features/FeaturesTest.java

```diff
@@ -167,9 +167,18 @@ public void getFamilyEmpty() throws Exception {
     Pair<String, String> p = Features.getFamily("");
   }
 
-  @Test(expected = RuntimeException.class)
-  public void getFamilyNotLABEL() throws Exception {
+  @Test
+  public void getDefaultStringFamily() throws Exception {
+    Pair<String, String> p = Features.getFamily("LABE");
+    assertEquals("", p.getLeft());
+    assertEquals(Features.DEFAULT_STRING_FAMILY, Features.getStringFamily(p));
+  }
+
+  @Test
+  public void getDefaultFloatFamily() throws Exception {
     Pair<String, String> p = Features.getFamily("LABE");
```

Comment:
no, this is just feature name, it was to throw, I can change it to a better name